sunpy/net/dataretriever/sources/kanzelhohe.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+import numpy as np
+
+import astropy.units as u
+
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+from sunpy.net import attrs as a
+
+__all__ = ['KanzelhoheClient']
+
+
+class KanzelhoheClient(GenericClient):
```

Comment:
This is more tricky... if we don't put `ö` on the class name, should we put `oe`? 

@segfaulthunter what would you say?